.circleci/config.yml

```diff
@@ -287,7 +282,37 @@ jobs:
       - checkout
       - *setup_remote_docker
       - *install_docker_client
-      - *docker_build
+      - run:
+          name: Build Docker image
+          command: |
+            ~/bin/docker build .
+
+  docker_push_tag:
+    <<: *container_config
+    steps:
+      - checkout
+      - *setup_remote_docker
+      - *install_docker_client
+      - run:
+          name: Build and push Docker image to DockerHub
+          command: |
+            ~/bin/docker login -u $DOCKER_USER -p $DOCKER_PASS
+            ~/bin/docker build -t ${DOCKERHUB_REPO:?}:${CIRCLE_TAG:?} .
+            ~/bin/docker tag ${DOCKERHUB_REPO:?}:latest ${DOCKERHUB_REPO:?}:${CIRCLE_TAG:?}
+            ~/bin/docker push ${DOCKERHUB_REPO:?}:${CIRCLE_TAG:?}
```

Comment:
How does the latest tag get pushed? Is a separate second distinct push needed or it gets pushed automagically with the related tag?

Reply:
I'm currently investigating this

Reply:
Should be fixed in 0fa66473